src/Console/Command/DispatchCommand.php

```diff
@@ -12,28 +12,37 @@
 namespace Sonata\DevKit\Console\Command;
 
 use Github\Exception\ExceptionInterface;
+use GitWrapper\GitWrapper;
 use Packagist\Api\Result\Package;
 use Sonata\DevKit\Config\Configuration;
+use Sonata\DevKit\Console\Style\SonataStyle;
 use Symfony\Component\Config\Definition\Processor;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Yaml\Yaml;
 
 /**
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  */
 class DispatchCommand extends Command
 {
+    const GITHUB_GROUP = 'sonata-project';
+    const GITHUB_USER = 'SonataCI';
+    const GITHUB_EMAIL = 'ci@sonata-project.org';
```

Comment:
@rande the PR is ready. All I need before merge is the real email of `SonataCI` user.

Could you please give me this one?